Rectangle.java

```diff
@@ -0,0 +1,35 @@
+
+public class Rectangle {
+	private int x;
+	private int y;
+	private int width;
+	private int height;
+	
+	public Rectangle(int x, int y, int w, int h) {
+		this.x = x;
+        this.y = y;
+        this.width = w;
+        this.height = h;
+	}
+	
+	public int getX() {
+		return this.x;
+	}
+	public int getY() {
+		return this.y;
+	}
+	public int getWidth() {
+		return this.width;
+	}
+	public int getHeight() {
+		return this.height;
+	}
+	
+	public String toString() {
+		return "Rectangle: { (" + this.x + ", " + this.y + "), " + this.width + "x" + this.height + " }";
```

Comment:
This is good but I would recommend using String.format() to construct strings with multiple variables. Easier to read.https://www.google.com/search?q=string+format+example&oq=String+format+example&aqs=chrome.0.0l6.4504j0j7&sourceid=chrome&ie=UTF-8